docs/topics/jobs.md

```diff
@@ -1,14 +1,9 @@
 # Jobs
 
-Looking for a Django REST Framework related job? We highlight job vacancies by our awesome sponsors here and also provide a list of other job resources that may be helpful.
+Looking for a new Django REST Framework related role? On this site we provide a list of job resources that may be helpful. It's also worth checking out if any of [our sponsors are hiring][drf-funding].
```

Comment:
Grand - looks good for first pass! We might punt it harder in the future. (Compromise for us might be to only feature our premium sponsors in this slot and just link to the others)